auction_keeper/main.py

```diff
@@ -292,8 +316,15 @@ def check_flop(self):
     def check_all_auctions(self):
         for id in range(1, self.strategy.kicks() + 1):
             with self.auctions_lock:
-                if self.check_auction(id):
+                if not self.check_auction(id):
+                    continue
+
```

Comment:
Might want to follow your pattern here and leave a single line comment saying:
`# current number of auctions don't exceed max auctions arg`